HACKING.md

```diff
@@ -17,19 +17,30 @@ In the following sections, the section header may indicate whether the
 section applies to the local cluster case (`[LOCAL]`) or the official
 prod case (`[PROD]`).
 
+You'll want to be sure you have KVM available in your cluster.  See
+[this section of the coreos-assembler docs](https://github.com/coreos/coreos-assembler/blob/master/README.md#getting-started---prerequisites).
+
+### [PROD] Using a production OpenShift 4 cluster
```

Comment:
`[PROD]` right now documents the process for setting up the current official pipeline. So let's leave that off until we actually change the setup.